libnavigation-ui/src/main/java/com/mapbox/navigation/ui/alert/AlertView.java

```diff
@@ -124,23 +149,21 @@ private void initAnimations() {
     slideDownTop.setInterpolator(new OvershootInterpolator(2.0f));
   }
 
-  private void initBackground() {
-    if (android.os.Build.VERSION.SDK_INT < android.os.Build.VERSION_CODES.LOLLIPOP) {
-      int progressColor = ThemeSwitcher.retrieveThemeColor(getContext(),
-        R.attr.navigationViewProgress);
-      int progressBackgroundColor = ThemeSwitcher.retrieveThemeColor(getContext(),
-        R.attr.navigationViewProgressBackground);
-
-      LayerDrawable progressBarDrawable = (LayerDrawable) alertProgressBar.getProgressDrawable();
-      // ProgressBar progress color
-      Drawable progressBackgroundDrawable = progressBarDrawable.getDrawable(0);
-      progressBackgroundDrawable.setColorFilter(progressBackgroundColor, PorterDuff.Mode.SRC_IN);
+  private void applyCustomizedAttributes() {
+    CardView alertCardView = findViewById(R.id.alertCardView);
```

Comment:
Since the View extends from `CardView` we don't need to do this. Also `setCardBackgroundColor` can be directly called without doing `alertCardView.setCardBackgroundColor()`

Reply:
yeah, that's what I was thinking and doing initially until testing shows it's wrong.